app/views/journeys/expenses/repairsandmaintenance/RepairsAndMaintenanceAmountView.scala.html

```diff
@@ -35,39 +35,35 @@
         businessId: BusinessId,
         accountingType: AccountingType)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("repairsAndMaintenanceAmount.title"))) {
-    
+@layout(pageTitle = title(form, messages(s"repairsAndMaintenanceAmount.heading.$userType"))) {
```

Comment:
Page title should now be '.heading.individual'

Reply:
done